android-lib/src/main/java/com/kinvey/android/async/AsyncBatchPushRequest.java

```diff
@@ -0,0 +1,267 @@
+/*
```

Comment:
since this is a new class, do you think we should implement in Kotlin instead? And Also `KinveyPushBatchResponse`, `KinveySyncSaveBatchResponse`, `KinveyUpdateObjectsResponse`, `KinveyUpdateSingleItemError`

Reply:
for `Kotlin` classes implementation,
Kotlin support updates currently only exists in `kotlin-migration` branch and that changes I think we cannot merge into `indev` branch because we need some tasks to do for full `Kotlin` support.
So currently I use `Java` for this classes. Maybe later we will convert all this classes to `Kotlin` but in this `kotlin-migration` branch then.